RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -200,6 +209,7 @@ struct ManageSubscriptionsButtonsView: View {
                 Task {
                     openURL(URLUtilities.createMailURL()!)
                 }
+                completionHandler.supportContacted()
```

Comment:
Ideally this calls the view model, which internally calls whatever it needs I think.